### PR TITLE
Skip replication when there are no changed chunks

### PR DESCRIPTION
**Pull request by Dany9966**

### CODE REVIEW

File: coriolis/providers/replicator.py [hunk 1]
```diff
@@ -795,27 +795,27 @@ def replicate_disks(self, source_volumes_info, backup_writer):
                 self._event_manager.progress_update(
                     "No new chunks to replicate for disk \"%s\" (%s)" % (
                         volume['disk_id'], devName))
```
Comment (aznashwan) [inline coriolis/providers/replicator.py:797]: Why not just add the following two lines here?
```python
self._repl_state = curr_state
return self._repl_state
```

**Review by aznashwan — CHANGES_REQUESTED** [review]
Can be simplified.

File: coriolis/providers/replicator.py (hunk 1 above)
Comment (Dany9966) [reply coriolis/providers/replicator.py:797]: Logically the same, but can do that too.

**Comment by aznashwan** [comment]
LGTM. Merging. Thanks!

**Pull request merged**